src/charm.py

```diff
@@ -255,6 +264,21 @@ def _common_install_code(self) -> bool:
 
         return True
 
+    def _setup_service(self, state: CharmState) -> None:
```

Comment:
nit: this method is called below, according to our standard it should appear lower in the file

Reply:
Updated.